ethcore/src/miner/miner.rs

```diff
@@ -564,7 +570,7 @@ impl MinerService for Miner {
 				for tx in &txs {
 					let _sender = tx.sender();
 				}
-				let _ = self.import_transactions(txs, |a| AccountDetails {
+				let _ = self.import_transactions(chain, txs, |a| AccountDetails {
```

Comment:
Sealing will be updated multiple times here:
1. Once for each retracted block + additional update sealing at the end of the function.